knative-operator/deploy/resources/networkpolicies.yaml

```diff
@@ -1,83 +0,0 @@
-apiVersion: networking.k8s.io/v1
```

Comment:
are these NetworkPolicies only there due to ServiceMesh? Honest question, I don't remember why we had them 😂

Reply:
Yes, as per the commit log https://github.com/openshift-knative/knative-serving-operator/commit/c67021b1fa43ba1fcc00f709cd27d9bfd944ac10, we added the NetworkPolicies because ServiceMesh introduced strict policy. Very nice commit log :joy: